vmhost/hostCore/execution.go

```diff
@@ -339,6 +339,8 @@ func (host *vmHost) ExecuteOnDestContext(input *vmcommon.ContractCallInput) (vmO
 		vmOutput, isChildComplete, err = host.executeOnDestContextNoBuiltinFunction(scExecutionInput)
 	}
 
+	host.Output().CompleteLogEntriesWithCallType("ExecuteOnDestContext")
```

Comment:
I do not think this is a good place to add - as you call this endpoint from asyncCall and after transferAndExecute as well.